python/glow/wgr/functions.py

```diff
@@ -19,6 +19,8 @@
 from typeguard import check_argument_types, check_return_type
 from typing import Dict, List
 
+__all__ = ['get_sample_ids', 'block_variants_and_samples', 'reshape_for_gwas']
```

Comment:
We probably need a newline before each requirements list; I'm not seeing a formatted list.

Reply:
Oh sorry, in the docstrings below. I just couldn't comment on the appropriate lines, eg:

```
 Requires that:
        - Each variant row has the same number of values
        - The number of values per row matches the number of sample IDs
```

These aren't actually made into a formatted list because I think there needs to be a leading newline.